backends/ebpf/psa/ebpfPsaGen.cpp

```diff
@@ -579,9 +579,11 @@ bool ConvertToEBPFControlPSA::preorder(const IR::TableBlock *tblblk) {
 }
 
 bool ConvertToEBPFControlPSA::preorder(const IR::AssignmentStatement *a) {
-    // the condition covers both ingress and egress timestamp
-    if (a->right->toString().endsWith("timestamp")) {
-        control->timestampIsUsed = true;
+    if (auto m = a->right->to<IR::Member>()) {
```

Comment:
I think you can do even better: just use the IR::Member visitor function and check there.